db/migrate/20181012160010_remove_special_characters_from_ansible_rabbitmq_password.rb

```diff
@@ -0,0 +1,24 @@
+require 'securerandom'
+
+class RemoveSpecialCharactersFromAnsibleRabbitmqPassword < ActiveRecord::Migration[5.0]
+  # used only in specs
+  class MiqDatabase < ActiveRecord::Base; end
+
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    auth = Authentication.find_by(
```

Comment:
Will there by multiple of these in the global, or do we not care?  Should we do .in_my_region?

Reply:
Well it shouldn't matter but we don't want these to get out of sync so I'll do the `.in_my_region`.

Reply:
It's not like the new password is deterministic so it would be particularly bad in this case